packages/frontend-2/lib/projects/composables/projectManagement.ts

```diff
@@ -268,24 +268,26 @@ export function useUpdateProject() {
 
   return async (
     update: ProjectUpdateInput,
+    customSuccessMessage?: string,
```

Comment:
Why not put this inside of `options`? That's what that's for. With this kind of backwards incompatible change (adding a new param before pre-existing params) you have to go through all usages of this function and make sure `options` is now the third parameter